packer.mk

```diff
@@ -0,0 +1,17 @@
+build-packer:
+	@packer build -var-file=packer/settings.json packer/vhd-image-builder.json
+
+init-packer:
+	@./packer/init-variables
+
+az-login:
+	az login --service-principal -u ${CLIENT_ID} -p ${CLIENT_SECRET} --tenant ${TENANT_ID}
+
+run-packer:
+	@packer version && make az-login && make init-packer && (make build-packer | tee packer-output)
+
+az-copy:
+	@make az-login && (azcopy --source "${OS_DISK_SAS}" --destination "${CLASSIC_BLOB}/${VHD_NAME}" --dest-sas "${CLASSIC_SAS_TOKEN}")

```

Comment:
I think `az login` should be a make target dependency instead:

```Makefile
az-copy: az-login
        azcopy --source "${OS_DISK_SAS}" --destination "${CLASSIC_BLOB}/${VHD_NAME}" --dest-sas "${CLASSIC_SAS_TOKEN}"
```

Same for the `generate-sas:` target below.

Reply:
Good point, I'll fix it. Thanks!